infra/base-images/base-runner/test_all

```diff
@@ -41,6 +41,9 @@ mkdir $TMP_FUZZER_DIR
 # Move contents of $OUT/ into $TMP_FUZZER_DIR. We can't move the directory
 # itself because it is a mount.
 mv $OUT/* $TMP_FUZZER_DIR
+INITIAL_OUT=$OUT
+OUT=$TMP_FUZZER_DIR
```

Comment:
don't you need to `export OUT=$tmp_fuzzer_dir` to make sure it's passed to the other script?

Reply:
Actual no, because `OUT` is already exported when we set it we are setting an exported variable.

Reply:
Sounds good then!

Reply:
I've changed these so that it doesn't confuse someone looking at this down the road.